Dockerfile.ecr

```diff
@@ -0,0 +1,23 @@
+FROM python:3.6-alpine
+
+RUN apk update && apk add postgresql-dev gcc python3-dev musl-dev bash git libffi-dev make
+
+ENV FLASK_ENV=production
+ENV REDIS_URL=redis://support-service-redis.be4fj4.ng.0001.usw2.cache.amazonaws.com:6379
```

Comment:
Can you make these unset variables. We can set them in the task definition. We don't want them hardcoded.